projects/create-and-vary-a-licence-and-delius/src/main/kotlin/uk/gov/justice/digital/hmpps/api/resource/StaffResource.kt

```diff
@@ -31,4 +32,9 @@ class StaffResource(
         staffService.findStaffForUsernames(
             usernames
         )
+
+    @PreAuthorize("hasRole('CVL_CONTEXT')")
+    @GetMapping("/{staffCode}/caseload/managed-offenders")
+    fun getManagedOffenders(@PathVariable staffCode: String): List<ManagedOffender> =
```

Comment:
The Community API version was paginated - do we need to do the same here?  Sorry if it's already been discussed.

Reply:
The current community API is not currently paginated. See https://community-api-public.test.probation.service.justice.gov.uk/swagger-ui/index.html#/Staff/getCaseloadOffendersForStaff - i discussed this with Anthony and we decided not to introduce this.

Reply:
My mistake - I was looking at the team caseload endpoint.